langchain4j-vertex-ai-gemini/src/test/java/dev/langchain4j/model/vertexai/VertexAiGeminiChatModelIT.java

```diff
@@ -390,6 +408,23 @@ void should_handle_parallel_function_calls() {
 
         assertThat(inventoryStock).containsIgnoringCase("ABC123");
         assertThat(inventoryStock).containsIgnoringCase("XYZ789");
+
+        // when
+        allMessages.add(messageResponse.content());
+
+        allMessages.add(ToolExecutionResultMessage.toolExecutionResultMessage(null, "getProductInventory",
+            "{\"product_id\":\"ABC123\", \"stock\": 10}"));
+        allMessages.add(ToolExecutionResultMessage.toolExecutionResultMessage(null, "getProductInventory",
+            "{\"product_id\":\"XYZ789\", \"stock\": 5}"));
+
+        messageResponse = model.generate(allMessages, stockInventoryToolSpec);
+
+        System.out.println("Final response = " + messageResponse.content().text());
+
+        // then
+        String text = messageResponse.content().text();
+        assertThat(text.split(" ")).containsAll(
```

Comment:
minor: hard to read/understand this assertion

Reply:
Right, I'll try to update it a bit to make it clearer.